service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -34,14 +35,42 @@
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.http.MediaType;
 import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
 
 @WebMvcTest
-@ContextConfiguration(classes = RunSetsApiController.class)
+@TestPropertySource(properties = "cbas.cbas-api.runSetsMaximumRecordIds=2")
```

Comment:
Oh this is cool! Does this let you use `records` again (since you don't need to mock it, presumably it being final is no longer a problem...? 🤞 )

Reply:
it doesn't seem to let me use `record`, sadly... there must be some piece that we're still missing to make that work.

this was Phil's suggestion -- he argued that configurations shouldn't be "mocked" as much as there should be a (real) configuration specifically for testing